pages/main-page.ts

```diff
@@ -0,0 +1,22 @@
+export class MainPage {
+  readonly page: Page;
+  constructor(page: Page) {
+    this.page = page;
+    this.setStorage = page.getByRole('link', { name: 'Storage' });
+    this.setUsers = page.getByRole('link', { name: 'Users' });
+    this.installButton = page.getByRole("button", { name: "Install", exact: true });
+    this.continueButton = page.getByRole("button", { name: "Continue" });
+  }
+  async set_storage() {
```

Comment:
don't forget camelcase, setStorage, this is not Perl :)

Reply:
updated